src/Components/SmartComponents/Systems/Systems.js

```diff
@@ -1,359 +1,206 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
please remove the comments that disable eslint errors. Especially `react-hooks/exhaustive-deps` is here to warn us. 

As for the `camel-case`   
you can assign a new variable when you extract it, for example `{ page, per_page: pageSize}` or if it comes from the backend and it needs to be snake_case just disable the eslint for this the particular line